### DISCUSSION TITLE
Node exist in Overpass but not exist in Nominatim

### DISCUSSION THREAD

**Discussion by dlubom** [osm-search/Nominatim#3306] [Q&A] [answered]
Hello,

I see this node on Overpass:

[out:json];
node(11237047754);
out center tags;

https://overpass-turbo.eu/s/1Ghu

But its not exists on Nominatim:

https://nominatim.openstreetmap.org/details.php?osmtype=N&osmid=11237047754&addressdetails=1&hierarchy=0&group_hierarchy=1&format=json

Question, why ?

**Comment by mtmail** [comment, marked as answer]
Nominatim merges two places together if they represent the same. In this case it already has https://nominatim.openstreetmap.org/ui/details.html?osmtype=W&osmid=1212880550&class=boundary

**Comment by lonvia** [comment]
The tagging of the node is unusual, which might also contribute to the problem. I've left a comment: https://www.openstreetmap.org/changeset/142117450
